extensions/database/src/com/google/refine/extension/database/SQLType.java

```diff
@@ -41,14 +42,24 @@ private SQLType(DriverContainer container) {
         this.driverContainer = container;
     }
 
-    public static SQLType forName(String name) {
-        for (SQLType sqlType : jdbcDriverRegistry.values()) {
-            if (sqlType.getIdentifier().equalsIgnoreCase(name)) {
-                return sqlType;
+    public static SQLType forName(List<String> names) {
+        for(String name : names){
+            for (SQLType sqlType : jdbcDriverRegistry.values()) {
+                if (sqlType.getIdentifier().equalsIgnoreCase(name)) {
+                    return sqlType;
```

Comment:
The signature of this method looks odd to me - is this actually used anywhere?

Reply:
<img width="374" alt="image" src="https://github.com/OpenRefine/OpenRefine/assets/51887332/d2b2d656-2781-4605-ac5a-77e6e824b231">

it is used to check if the jdbc driver is present in `jdbcDriverRegistry` map.

<img width="795" alt="image" src="https://github.com/OpenRefine/OpenRefine/assets/51887332/72604f38-2f60-45a8-a6c4-a79f98a9de1c">


same is being used in other vendor-specific classes.